src/OracleService/Settings.cs

```diff
@@ -14,13 +14,24 @@ public HttpsSettings(IConfigurationSection section)
         }
     }
 
+    class FsSettings
```

Comment:
One file, one class?

Reply:
It's a part of settings. I think it's okay.